### DISCUSSION TITLE
self-propelled rods simulations

### DISCUSSION THREAD

**Discussion by activevsesi** [glotzerlab/hoomd-blue#1854] [Support] [answered]
Hi HOOMD-blue users,
First of all thanks for providing the open source code for the community. I am quite new to the hood blue. We are interested in studying the dynamic self-organization of self-propelled or active rods for different concentrations, etc. Is there a way to use HOOMD-blue for active anisotropic particles? Any pointers would be much appreciated. thanks.

**Comment by joaander** [comment, marked as answer]
Yes, use [`force.Active`](https://hoomd-blue.readthedocs.io/en/v4.8.1/module-md-force.html#hoomd.md.force.Active).

**Reply by activevsesi** [reply]
Thank you very much.
